Dockerfile

```diff
@@ -1,14 +1,112 @@
-ARG ACAS_IMAGE=mcneilco/acas-oss:latest
-FROM ${ACAS_IMAGE} AS acas-src
+FROM centos:centos7
 
-FROM mcneilco/centos-r-repo:XLConnect
-COPY	. /home/runner/racas
+# R, Java
+ENV JAVA_VERSION 1.8.0
+ENV JAVA_HOME /usr/lib/jvm/java
+ENV LD_LIBRARY_PATH $JAVA_HOME/jre/lib/amd64:$JAVA_HOME/jre/lib/amd64/server/
+RUN echo "$JAVA_HOME/jre/lib/amd64/server/" > /etc/ld.so.conf.d/rApache_rJava.conf && \
+    /sbin/ldconfig
+
+RUN \
+  useradd -u 1001 -ms /bin/bash builder && \
+  yum update -y && \
+  yum upgrade -y && \
+  yum install -y epel-release && \
+  yum install -y rpm-build which make wget tar httpd-devel libapreq2-devel libcurl-devel protobuf-devel openssl-devel libpng-devel java-1.8.0-openjdk java-1.8.0-openjdk-devel && \
+  yum-builddep -y R-devel && \
+  yum clean all
+
+# R version
+# https://github.com/rstudio/r-docker/blob/master/3.2/centos7/Dockerfile
+# Can't use because RPM R not build as shared library which rapache needs
+# ENV R_VERSION 3.2.3
+# RUN curl -O https://cdn.rstudio.com/r/centos-7/pkgs/R-${R_VERSION}-1-1.x86_64.rpm && \
+#     yum -y install ./R-${R_VERSION}-1-1.x86_64.rpm && \
+#     ln -s /opt/R/${R_VERSION}/bin/R /usr/bin/R && \
+#     ln -s /opt/R/${R_VERSION}/bin/Rscript /usr/bin/Rscript && \
+#     ln -s /opt/R/${R_VERSION}/lib/R /usr/lib/R && \
+#     rm R-${R_VERSION}-1-1.x86_64.rpm && \
+#     yum clean all
+
+ENV R_VERSION 3.6.3
+RUN curl -O https://cran.rstudio.com/src/base/R-3/R-${R_VERSION}.tar.gz && \
+    tar -xzvf R-${R_VERSION}.tar.gz && \
+    cd R-${R_VERSION} && \
+    ./configure \
+        --prefix=/opt/R/${R_VERSION} \
+        --enable-memory-profiling \
+        --enable-R-shlib \
+        --with-blas \
+        --with-lapack && \
+    make && \
+    make install
+
+RUN ln -s /opt/R/${R_VERSION}/bin/R /usr/local/bin/R
+RUN ln -s /opt/R/${R_VERSION}/bin/Rscript /usr/local/bin/Rscript
+
+ENV JAVA_HOME /usr/lib/jvm/java-1.8.0
+ENV LD_LIBRARY_PATH $JAVA_HOME/jre/lib/amd64:$JAVA_HOME/jre/lib/amd64/server/
+RUN echo "$JAVA_HOME/jre/lib/amd64/server/" > /etc/ld.so.conf.d/rApache_rJava.conf && \
+    /sbin/ldconfig && \
+    R CMD javareconf
+
+USER builder
+
+RUN \
+  mkdir -p ~/rpmbuild/SOURCES && \
+  mkdir -p ~/rpmbuild/SPECS
+
+RUN \
+  cd ~ && \
+  wget https://github.com/jeffreyhorner/rapache/archive/v1.2.9.tar.gz -O rapache-1.2.9.tar.gz && \
```

Comment:
nit: might make sense for the rapache version to be an environment variable too.